lerna.json

```diff
@@ -21,5 +20,10 @@
         "test/**"
       ]
     }
-  }
+  },
+  "packages": [
+    "packages/*"
```

Comment:
Why is this needed? Lerna should use workspaces field from `package.json` when `useWorkspaces` flag is passed

Reply:
This is here because this PR is from last month when Lerna didn't support this yet AFAIR.